src/commands/network/networkGetConfig.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  ARTIFACTS_PATH,
+  loadNeverminedConfigContract,
+  StatusCodes
+} from '../../utils'
+import { Logger } from 'log4js'
+import { Account, Nevermined } from '@nevermined-io/nevermined-sdk-js'
+import { ExecutionOutput } from '../../models/ExecutionOutput'
+import * as fs from 'fs'
+import chalk from 'chalk'
+import { ConfigEntry } from '../../models/ConfigDefinition'
+import { ethers } from 'ethers'
+
+export const networkGetConfig = async (
```

Comment:
maybe rename this to `neverminedContractConfig`. Right now we have different configs: the nevermined instance config which is also related to the network, and the nevermined config contract. It's important that the names don't raise confusion